src/test/java/org/takes/rq/multipart/RqMtFakeTest.java

```diff
@@ -343,6 +346,29 @@ public void returnsCorrectNamesSet() throws IOException {
         }
     }
 
+    /*
+     * @todo #577:30min The test below fails with the message
+     *  "header "Content-Disposition" is mandatory".
+     *  The error disappears when `new RqFake(new ListOf<>(""), "")` is
+     *  changed to `new RqFake(new ListOf<>(""), "someValue")`.
+     *  Fix the underlying problem and remove the `@Ignore` annotation.
+     * */
+    @Test
+    @Ignore("See puzzle above")
+    public void contentDispositionShouldBeRecognized() throws IOException {
+        new RqMtFake(
+            new RqFake(),
+            new RqWithHeader(
+                new RqFake(new ListOf<>(""), ""),
+                contentDispositionHeader("form-data; name=\"field1\"")
```

Comment:
@Serranya Please, change this private method for one "more OO", like `FormattedText`.